cmake/kokkos_enable_devices.cmake

```diff
@@ -57,9 +57,9 @@ IF (KOKKOS_ENABLE_OPENMPTARGET)
     XL    KOKKOS_IBM_XL_OMP45_WORKAROUND
     Clang KOKKOS_WORKAROUND_OPENMPTARGET_CLANG
   )
-  COMPILER_SPECIFIC_LIBS(
-    Clang -lopenmptarget
-  )
+#  COMPILER_SPECIFIC_LIBS(
```

Comment:
Not sure why this every worked?

Reply:
"Why" or "if"?

Any reason to comment rather than just remove these lines?